src/Microsoft.OData.Core/Json/ODataUtf8JsonWriter.cs

```diff
@@ -86,7 +99,10 @@ internal sealed class ODataUtf8JsonWriter : IJsonWriter, IDisposable, IJsonWrite
         /// an array e.g. ["rawValue1", "rawValue2", "rawValue3"
         /// </summary>
         private bool isWritingConsecutiveRawValuesAtStartOfArray = false;
-        
+        /// <summary>
+        /// Rrepresents a read-only memory block containing the byte representation of the double quote character ("). 
+        /// </summary>
+        private readonly static ReadOnlyMemory<byte> _doubleQuote = new byte[] { (byte)'"' };

```

Comment:
nit: We use `this.` instead of `_` for fields. Maybe for constants and statics we should be consistent with `PascalCase`
```suggestion
        private readonly static ReadOnlyMemory<byte> doubleQuote = new byte[] { (byte)'"' };

```

Reply:
I think since it's a static you can't use `this.` I remove the static qualifier then.